### PR TITLE
demo: realistic output

### PR DESCRIPTION
**Pull request by yuindustries**
output-demo

### CODE REVIEW

File: cmd/tiltdemo/main.go [hunk 1]
```diff
@@ -9,26 +9,67 @@ import (
...
 
 	awaitInput()
-
 	println(100, "Building [my-app-backend]")
-	println(500, "Build complete in 0.1232s")
+	printlnColor(200, "Build complete in 1.2342s")
 
 	awaitInput()
-
 	println(100, "Building [my-app-backend2]")
```
Comment (dbentley) [inline cmd/tiltdemo/main.go]: Do we expect to print the build output for later builds?

File: cmd/tiltdemo/main.go [hunk 2]
```diff
@@ -9,26 +9,67 @@ import (
...
-	println(500, "Build complete in 1.3532s")
+	printlnColor(200, "Build complete in 2.1332s")
```
Comment (dbentley) [inline cmd/tiltdemo/main.go]: We probably need better terminology, as it's also pushing.

File: cmd/tiltdemo/main.go [hunk 3]
```diff
@@ -9,26 +9,67 @@ import (
...
+	println(100, " → applying via kubectl")
+	println(100, "Successfully built 2917b4065035")
+	println(100, "Successfully tagged 2917b4065035")
+	printlnColor(200, "Build complete in 7.2342s")
 
 	awaitInput()
-
 	println(100, "Building [my-app-backend]")
-	println(500, "Build complete in 0.1232s")
+	printlnColor(200, "Build complete in 1.2342s")
```
Comment (dbentley) [inline cmd/tiltdemo/main.go]: It'd be great to include how much was their build, vs. the rest.

File: cmd/tiltdemo/main.go [hunk 4]
```diff
@@ -9,26 +9,67 @@ import (
...
+	newline()
+	println(100, "Parsing Kubernetes config YAML")
+	println(100, " → applying via kubectl")
+	println(100, "Successfully built 2917b4065035")
+	println(100, "Successfully tagged 2917b4065035")
+	printlnColor(200, "Build complete in 7.2342s")
 
 	awaitInput()
-
 	println(100, "Building [my-app-backend]")
```
Comment (dbentley) [inline cmd/tiltdemo/main.go]: We should say what file was edited that's causing us to rebuild.

**Comment by yuindustries** [comment]
tilt-ideal

File: cmd/tiltdemo/main.go [hunk 5]
```diff
@@ -8,27 +8,101 @@ import (
...
+	printlnColor(200, "  → Done in 1.234s", cBlue)
+
+	newline()
+	printlnColor(100, "Deploying: [devel-hanyu-lb-blorg-be]", cGreen)
+	println(100, "  → parsing config YAML")
+	println(100, "  → applying config via kubectl")
+	println(100, "  → service created")
+	println(100, "  → deployment created")
+	printlnColor(200, "  → Done in 472ms", cBlue)
+	newline()
```
Comment (dbentley) [inline cmd/tiltdemo/main.go]: I think my big remaining question is about terminology and timing for the overall pipeline run.

A user should see how long the entire thing took, and how much of that was their build (which we don't control), as a summary, without having to scroll.

**Comment by yuindustries** [comment]
color code output

**Comment by yuindustries** [comment]
color code output 2

**Pull request merged**